assets/js/modules/analytics/components/common/ProfileSelect.test.js

```diff
@@ -50,6 +59,10 @@ const setupRegistryWithExistingTag = ( { dispatch } ) => {
 		propertyID: fixtures.accountsPropertiesProfiles.profiles[ 0 ].webPropertyId, // eslint-disable-line sitekit/camelcase-acronyms
 	};
 	const { id } = fixtures.propertiesProfiles.profiles[ 0 ];
+
+	dispatch( STORE_NAME ).finishResolution( 'getAccounts', [] );
+	dispatch( STORE_NAME ).finishResolution( 'getProperties', [ existingTag.accountID ] );
+	dispatch( STORE_NAME ).finishResolution( 'getProfiles', [ existingTag.accountID, existingTag.propertyID ] );
```

Comment:
See above, let's also populate data for these selectors.

Reply:
Updated.